spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'logger'
+
+class ParentClass
```

Comment:
We shouldn't declare a class in the global context, I think you can accomplish this with an rspec mock
```
vpc = double("IBM::Cloud::SDK::Vpc")
allow(vpc).to receive(:logger).and_return(Logger.new(nil))
allow(vpc).to receive(:url)
etc...
```

Reply:
Ahh yes, hadn't thought of that. I'll try it.